src/components/Input.js

```diff
@@ -63,21 +62,13 @@ const Input = props => {
         onChangeProp(`${month}/${day}/${(newValue || value).getFullYear()}`, name, e);
         close();
       } else if (type === 'numeric') {
-        onChangeProp(
-          /^[0-9]+$/.test(trim ? e.trim() : e)
-            ? trim
-              ? e.trim()
-              : e
-            : e.substring(0, e.length - 1),
-          name,
-          e,
-        );
+        onChangeProp(/^[0-9]+$/.test(e.trim()) ? e.trim() : e.substring(0, e.length - 1), name, e);
```

Comment:
@yonngNessa14 did you make this change or did Hrach?

Reply:
Hrach did, not me @cjayprime